### DISCUSSION TITLE
CSV or OCB file doesn't open in OpenChrom

### DISCUSSION THREAD

**Discussion by octavianparascinet** [OpenChrom/openchrom#475] [Q&A]
Hi everyone! 
I have a problem here. I'm using OpenChrom for analyzing some chromatograms and identifying/quantifying their peaks. Everything goes normally with the manual detection peaks (I've used Peak Detector UI) and the integration, but when I save the chromatogram in csv or ocb format it doesn't open in the app. I don't know how to recover all the information (detection and integration of peaks) from the saved files.

**Comment by eselmeister** [comment]
Which OpenChrom version do you use?
Could you supply a *.ocb file?

Am 10.08.21 um 14:55 schrieb octavianparascinet:
>
> Hi everyone!
> I have a problem here. I'm using OpenChrom for analyzing some 
> chromatograms and identifying/quantifying their peaks. Everything goes 
> normally with the manual detection peaks (I've used Peak Detector UI) 
> and the integration, but when I save the chromatogram in csv or ocb 
> format it doesn't open in the app. I don't know how to recover all the 
> information (detection and integration of peaks) from the saved files.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/OpenChrom/openchrom/issues/130>, or unsubscribe 
> <https://github.com/notifications/unsubscribe-auth/AAFHUT2KAGYMCF2BGSHUTJLT4EOVBANCNFSM5B4DNGYQ>.
> Triage notifications on the go with GitHub Mobile for iOS 
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> 
> or Android 
> <https://play.google.com/store/apps/details?id=com.github.android&utm_campaign=notification-email>.
>

-- 
~~~~~~~~~~~~~~~~~~~~~~~~
OpenChrom - the open source alternative for chromatography / mass spectrometry
Dr. Philip Wenig » Founder » ***@***.*** » http://www.openchrom.net
~~~~~~~~~~~~~~~~~~~~~~~~

**Comment by octavianparascinet** [comment]
The latest version available on Lablicate's page, 1.4.x. Ok, this is one of the files that I have saved after I did the peak detection. I send it to you in csv format because doesn't allow me to attach it in ocb. 
[1B.csv](https://github.com/OpenChrom/openchrom/files/6966806/1B.csv)

**Comment by Mailaender** [comment]
Try to open and save the file, then go to `Logging` and upload you `ChemClipse.log`.

**Comment by eselmeister** [comment]
@octavianparascinet Thanks for providing the demo file. I'm able to load it:

![Bildschirmfoto von 2021-08-12 15-32-11](https://user-images.githubusercontent.com/686671/129205870-bd714656-6726-45f9-bdbd-a71bab15aee8.png)

Could you send me a screenshot of your version?

**Comment by octavianparascinet** [comment]
Ok, but how do I get back all the detected peaks with their RT and their percentage area? Because it does open to me as well, as you see in the screenshot I attached. But there's no information about these already measured parameters. 
<img width="1440" alt="Captura de pantalla 2021-08-16 a las 13 35 19" src="https://user-images.githubusercontent.com/88721400/129557655-b7b317c2-f2c9-4420-a14d-0e3089163579.png">

**Comment by Mailaender** [comment]
That information is not saved in `.csv` but in `.ocb` files.

**Comment by octavianparascinet** [comment]
And how do I read the .ocb file in OpenChrom. Nothing appears on the interface when I do so. Sorry I'm inexperienced with this software and I need a little help. Thank you for your replies!

**Comment by eselmeister** [comment]
Please do the following:

* open a csv file with your data
* run a peak detection / integration on your chromatogram
* save as... the chromatogram in *.ocb format
* close the chromatogram
* open the *.ocb chromatogram

Please create screenshots of all the above mentioned steps.

Am 30.08.21 um 13:06 schrieb octavianparascinet:
>
> And how do I read the .ocb file in OpenChrom. Nothing appears on the 
> interface when I do so. Sorry I'm inexperienced with this software and 
> I need a little help. Thank you for your replies!
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub 
> <https://github.com/OpenChrom/openchrom/issues/130#issuecomment-908251012>, 
> or unsubscribe 
> <https://github.com/notifications/unsubscribe-auth/AAFHUT3SEEJYMHJM2LP2JXDT7NQ33ANCNFSM5B4DNGYQ>.
> Triage notifications on the go with GitHub Mobile for iOS 
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> 
> or Android 
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>. 
>
>

-- 
~~~~~~~~~~~~~~~~~~~~~~~~
OpenChrom - the open source alternative for chromatography / mass spectrometry
Dr. Philip Wenig » Founder » ***@***.*** » http://www.openchrom.net
~~~~~~~~~~~~~~~~~~~~~~~~
